templates/events-my-events.php

```diff
@@ -34,10 +35,16 @@
 			$event_status                  = get_post_status( $event_id );
 			$event_start                   = ( new DateTime( get_post_meta( get_the_ID(), '_event_start', true ) ) )->format( 'M j, Y' );
 			$event_end                     = ( new DateTime( get_post_meta( get_the_ID(), '_event_end', true ) ) )->format( 'M j, Y' );
+			$envent_end_utc                = new DateTime( get_post_meta( get_the_ID(), '_event_end', true ), new DateTimeZone( 'UTC' ) );
+			$now                           = new DateTime( 'now', new DateTimeZone( 'UTC' ) );
+
+			$now > $envent_end_utc ? $past_event = true : $past_event = false;
```

Comment:
I was confused by this initially, I thought it had no purpose as the result of the comparison wasn't being used, but then I noticed variables are being assigned in the condition. Personally, I think it would be more readable to do something like:

```php
$past_event = false;
if ( $now > $event_end_utc ) {
    $past_event = true;
}
```

Reply:
As the `$past_event` is only used in one check, I removed it and put the comparison directly in the check.